exchange/adapter_map.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 
 	"github.com/prebid/prebid-server/adapters"
-	ttx "github.com/prebid/prebid-server/adapters/33across"
+	"github.com/prebid/prebid-server/adapters/33across"
```

Comment:
Need to keep the `ttx` part or you will break the line below that access the adapter by this alias